src/condor_tools/htcondor_cli/annex_create.py

```diff
@@ -391,7 +494,8 @@ def annex_create(
     # so even if it's wrong, it will at least consistently so.
     username = getpass.getuser()
 
-    if target not in MACHINE_QUEUE_MAP:
+    target = target.lower()
```

Comment:
Nitpick (and this is what I should have done before), this is what [`str.casefold()`](https://docs.python.org/3.6/library/stdtypes.html#str.casefold) is for:
```suggestion
    target = target.casefold()
```